Makefile

```diff
@@ -633,12 +633,26 @@ fmt: ## Format the code using Ruff and blacken-docs
 	python -m ruff check --fix-only
 	python -m ruff format
 	@echo "Formatting the code blocks with blacken-docs..."
-	git ls-files -z -- '*.md' | xargs -0 blacken-docs -t=py39
+	@if [ "$(MLRUN_PYTHON_VERSION)" = "3.9" ]; then \
+	    git ls-files -z -- '*.md' | xargs -0 blacken-docs -t=py39; \
+	elif [ "$(MLRUN_PYTHON_VERSION)" = "3.11" ]; then \
+	    git ls-files -z -- '*.md' | xargs -0 blacken-docs -t=py311; \
+	else \
+	    echo "Unsupported Python version: $(MLRUN_PYTHON_VERSION)"; \
+	    exit 1; \
+	fi
 
 .PHONY: lint-docs
 lint-docs: ## Format the code blocks in markdown files
 	@echo "Checking the code blocks with blacken-docs"
-	git ls-files -z -- '*.md' | xargs -0 blacken-docs -t=py39 --check
+	@if [ "$(MLRUN_PYTHON_VERSION)" = "3.9" ]; then \
+	    git ls-files -z -- '*.md' | xargs -0 blacken-docs -t=py39 --check; \
+	elif [ "$(MLRUN_PYTHON_VERSION)" = "3.11" ]; then \
+	    git ls-files -z -- '*.md' | xargs -0 blacken-docs -t=py311 --check; \
+	else \
+	    echo "Unsupported Python version: $(MLRUN_PYTHON_VERSION)"; \
+	    exit 1; \
+	fi
```

Comment:
you can do this in 1-2 liner
see this example
```
# expected to be in the form of '-py<major><minor>' e.g. '-py39'
MLRUN_ANACONDA_PYTHON_DISTRIBUTION := $(shell echo "$(MLRUN_PYTHON_VERSION)" | awk -F. '{print "-py"$$1$$2}')
```
Calculate the env var you need and use it in the command
e.g.
```
MLRUN_LINT_PYTHON_VERSION=... # will be py39 or py311
git ls-files -z -- '*.md' | xargs -0 blacken-docs -t="${MLRUN_LINT_PYTHON_VERSION}" --check;
```